deps-mvn-indexing/src/main/clojure/com/connexta/osgeyes/index/core.clj

```diff
@@ -67,4 +68,76 @@
   (count (gather-hierarchy "ddf" "2.19.5"))
   (map #(artifact-info->map % #{:attrs}) (do-gather-hierarchy "ddf" "ddf" "2.19.5"))
   (map #(.toString %) (do-gather-hierarchy "ddf" "ddf" "2.19.5"))
-  (close-indexer!))
\ No newline at end of file
+  (close-indexer!))
+
+(def test-data
+  '({:artifact-id "a"
+     :attrs       {"JAR_PACKAGES" "package_1,package_2"}}
+    {:artifact-id "b"
+     :attrs       {"JAR_PACKAGES" "package_2,package3"}})
+  )
+
+(comment
```

Comment:
When you're done go ahead and clean up the comments; you can delete any that were already there as well.